actor/builtin/paymentbroker/paymentbroker.go

```diff
@@ -709,3 +735,19 @@ func findByChannelLookup(ctx context.Context, storage exec.Storage, byPayer exec
 
 	return actor.LoadTypedLookup(ctx, storage, byChannelCID, &PaymentChannel{})
 }
+
+func checkCondition(vmctx exec.VMContext, condition *types.Predicate, redeemerSuppliedParams []interface{}) (uint8, error) {
+	if condition == nil {
+		return 0, nil
+	}
+	params := append(condition.Params[:0:0], condition.Params...)
```

Comment:
huh cool, I erroneously thought the python slice copy: `params := condition.Params[:]` would work in go.

Reply:
Yeah, we had to research this too.